bibernate-core/src/main/java/com/bobocode/petros/bibernate/session/DefaultSession.java

```diff
@@ -8,11 +8,12 @@
 import javax.sql.DataSource;
 import java.util.Collection;
 import java.util.List;
+import java.util.Optional;
 
 public class DefaultSession implements Session {
 
-    private final DataSource dataSource;
-    private final JdbcQueryManager jdbcQueryManager;
+    protected final DataSource dataSource;
```

Comment:
If we call super.<some_operation> from cacheable session, then these modifiers may be private.

Reply:
Looks like we don't need a Datasource in Session at all ) JdbcQueryMannage is fully responsible for any manipulations with datasource